src/sagemaker/cli/compatibility/v2/modifiers/serde.py

```diff
@@ -157,6 +157,48 @@ def modify_node(self, node):
         )
 
 
+class SerdeKeywordRemover(Modifier):
+    """A class to remove Serde-related keyword arguments from call expressions."""
+
+    def node_should_be_modified(self, node):
+        """Checks if the ``ast.Call`` node uses deprecated keywords.
+
+        In particular, this function checks if:
+
+        - The ``ast.Call`` represents the ``create_model`` method.
+        - Either the serializer or deserializer keywords are used.
+
+        Args:
+            node (ast.Call): a node that represents a function call. For more,
+                see https://docs.python.org/3/library/ast.html#abstract-grammar.
+
+        Returns:
+            bool: If the ``ast.Call`` contains keywords that should be removed.
+        """
+        if not isinstance(node.func, ast.Attribute) or node.func.attr != "create_model":
+            return False
```

Comment:
optional - `create_model` is pretty generic, so you may want to check for common estimator variable names to avoid unintended collisions (e.g. https://github.com/aws/sagemaker-python-sdk/blob/zwei/src/sagemaker/cli/compatibility/v2/modifiers/renamed_params.py#L242)

however, `serializer` and `deserializer` are quite specific so maybe there's no need to worry here